docs/interacting/web-ui.md

```diff
@@ -0,0 +1,35 @@
+# Web application
+
+The web application is hosted at [skywalking-banyandb-webapp](http://localhost:17913/) when you boot up the BanyanDB server.
+The web ui could use to query data stored in streams, measures, and properties.
+
```

Comment:
The first section should be the dashboard. You could copy the [screenshot](https://github.com/apache/skywalking-banyandb/blob/main/docs/installation/cluster.md#self-observability-dashboard) here